### DISCUSSION TITLE
What exactly get published as update site ?

### DISCUSSION THREAD

**Discussion by UrsZeidler** [eclipse-birt/birt#1086] [General]
What exactly get published as update site from all these artifacts produced in a ci run (for example: https://github.com/eclipse/birt/actions/runs/3159947085)

I'm looking for the **org.eclipse.birt.rcp** feature which in included in the org.eclipse.birt.p2updatesite but not the published updatesite at https://download.eclipse.org/birt/update-site.

greetings, Urs.

**Comment by speckyspooky** [comment]
Check the following link, there you will find the current eclipse- and RCP-designer-versions of BIRT:
https://download.eclipse.org/birt/downloads/drops/
